openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip/swagger.yaml

```diff
@@ -36,6 +36,23 @@ security: # Default security on all endpoints without explicit declarations
       - write:pets
       - read:pets
 paths:
+  x-tapir-codegen-security-path-prefixes:
```

Comment:
The only two sensible places I can think of for this to go are here, and at the root level (would be annoying to require it on every path, since it's probably gonna be heavily duplicated). Both are valid extension sites according to the openapi specs, but scoping it to paths feels the more appropriate of the two